plugins/rtmp-services/data/services.json

```diff
@@ -2622,6 +2622,20 @@
                 "max video bitrate": 5000,
                 "max audio bitrate": 160
             }
-        }
+        },
+        {
+            "name": "Modern Stream",
+            "servers": [
+                {
+                    "name": "Default",
+                    "url": "rtmps://ingest.modern-stream.com:443/live"
+                }
+            ],
+            "recommended": {
+                "keyint": 2,
+                "x264opts": "tune=zerolatency",
+				"max fps": 30
+            }
+		}
```

Comment:
Please don't mix tabs and spaces, try to use spaces everywhere.

Also, are you sure you want to use `tune=zerolatency`? It significantly degrades visual clarity while only improving end-to-end latency by a very small amount.

Reply:
Okay, I will remove the "tune=zerolatency" and fix the indentation problem